server/src/main/java/org/opensearch/index/search/stats/SearchStats.java

```diff
@@ -141,7 +141,7 @@ private Stats(StreamInput in) throws IOException {
             suggestTimeInMillis = in.readVLong();
             suggestCurrent = in.readVLong();
 
-            if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
+            if (in.getVersion().onOrAfter(Version.V_2_3_0)) {
```

Comment:
Is this the only place we are doing serde and relying on versioning?

Reply:
https://github.com/opensearch-project/OpenSearch/pull/4616/files#diff-ea76cbb06bdfab2ac2c0a3f2f044be1c4f01f368d733af690156f18a31ca03b7R3

https://github.com/opensearch-project/OpenSearch/pull/4616/files#diff-ea76cbb06bdfab2ac2c0a3f2f044be1c4f01f368d733af690156f18a31ca03b7R99

These two test files ^

and searchStats file.